### ISSUE TITLE
Add an unofficially reproduced checkpoint for quick evaluation

### ISSUE THREAD

**Issue by Jiawei-Yang** [NVlabs/GaussianSTORM#1]
I’ve re-trained a model using publicly available resources that replicates the setup from the paper. While this checkpoint is not officially affiliated with NVIDIA, it can be a useful starting point for quick evaluation.

The checkpoint is at [here](https://drive.google.com/file/d/1SNBEEB0sreW22IXKMtrfIe7JG5ED29Eu/view?usp=drive_link).

Best,
Jiawei

**Comment by hungdche** [comment]
Thank you for releasing the unofficial checkpoint! When I ran inference, the output seems to have some color tone issues. I am wondering if it is an issue on my side only

https://github.com/user-attachments/assets/ce3ce7a9-d367-42bd-892c-1f176c1eb442

**Comment by Jiawei-Yang** [comment]
Good catch! It was due to the sigmoid activation function [here](https://github.com/NVlabs/GaussianSTORM/blob/4ecc17a66235061a3f4800e9d98f05d6bf14a337/storm/models/storm.py#L40). I'll also test this soon.

**Comment by kevinchiu19** [comment]
Hi, thank you for your wonderful work and open source code！

I refer to your training script and start training from scratch, and find that the evaluation results are always difficult to reach the results of this pre-trained weight.

Like this:

![Image](https://github.com/user-attachments/assets/18cd853c-177e-40e1-9948-7c0d63726a31)

I would like to ask what you think is the reason? Does the batchsize have to be 64? (I use 4 batches and one gpu here), or is there any other reason?

Thanks!

**Comment by Jiawei-Yang** [comment]
Yes, the batch size is important to achieve the same amount of computation used for training. I'm attaching a training log for reference:

<img width="1467" alt="Image" src="https://github.com/user-attachments/assets/6888c931-3622-4552-ae7f-7d894e8bb47e" />

**Comment by kevinchiu19** [comment]
Thank you for your quick reply. I will refer to the training log you provided for analysis.

**Comment by kevinchiu19** [comment]
Hi, I tried a experiment with 32 global batchsize (4 gpu x 8 batchsize) and found that the dynamic results did not improve. Please confirm whether your global batchsize is 64 (8 gpu x 4 batchsize x 2 nodes)?

![Image](https://github.com/user-attachments/assets/4f8acf40-0f83-477c-be6b-e5c9074abc32)

![Image](https://github.com/user-attachments/assets/3f03e4a0-07bc-476a-b5b6-90404cc21023)


Also, can you provide the training script and log txt file?

Thanks!

**Comment by Jiawei-Yang** [comment]
You can use these as a reference.
[args.json](https://github.com/user-attachments/files/20921640/args.json)
[training_metrics.json](https://github.com/user-attachments/files/20921639/training_metrics.json)
[log.txt](https://github.com/user-attachments/files/20921812/log.txt)

My effective batch size: 64

I think the results are quite close? Your flow evaluation part might have some issues.

**Comment by kevinchiu19** [comment]
Thank you for the information you provided. I re-ran the evaluation script and found that the results were slightly different from those after training.
The results of the re-run show that the effect is indeed close.

![Image](https://github.com/user-attachments/assets/d2e4fbac-bb54-422d-8497-95a1fa96b071)

I will check if there is any code bug in it.

Thank you again for your patience and quick response!
